src/2_RTB/PianoDiQualifica/content.typ

```diff
@@ -0,0 +1,153 @@
+        - Formula: $"PV" = "Percentuale di pianificazione del lavoro" dot "BAC"$.
+  - *AC*: _Actual Cost_ Il costo effettivo sostenuto fino a un dato momento;
+  - *CPI*:_Cost Performance Index_ Misura l'efficienza del costo del lavoro svolto fino a un dato momento;
+          - Formula: $"CPI" = "EV" / "AC"$.
+  - *SPI*: _Schedule Performance Index_ Misura l'efficienza del tempo rispetto alla pianificazione;
+        - Formula: $"SPI" = "EV" / "PV"$.
+  - *EAC*: _Estimated at Completion_ Stima del costo totale previsto per completare il progetto;
+        - Formula: $"EAC" = "BAC" / "CPI"$.
+  - *ETC*: _Estimated To Completion_ Stima del costo aggiuntivo necessario per completare il progetto;
+        - Formula: $"ETC" = "EAC" - "AC"$.
+  - *VAC*: _Variance at Completion_ La differenza tra il budget previsto e il budget attuale alla fine del progetto;
+        - Formula: $"VAC" = "BAC" - "EAC"$.
+  - *SV*:_Schedule Variance_ Indica se si è in linea, in anticipo o in ritardo rispetto alla schedulazione delle attività di progetto pianificate nella baseline;
+        - Formula: $"SV" = "EV" - "PV"$.
+  - *CV*:_Cost Variance_ La differenza tra il valore del lavoro effettivamente svolto e il costo effettivo del lavoro svolto fino a un dato momento;
+        - Formula: $ "CV" = "EV" - "AC"$.
+  - *BV*:_Budget Variance_ Indica se alla data corrente si è speso di più o di meno rispetto a quanto previsto a budget alla data corrente;
+        - Formula: $"BV" = "PV" - "AC"$.
+
+
+Le metriche della qualità di processo verrano identificate con:
+
+                     *MPC-[Nome abbreviato]*
+
+*Leggenda*:
+
+      - *Nome abbreviato*: Nome della metrica specifica abbreviata.
+
+#table(
+      columns:(auto,auto,auto,auto),
+      align: (x, y) => (center, center, center,center).at(x),
+      fill:(_,row) =>if row==0 {luma(150)}, else if calc.odd(row) { luma(220)} else {white},
+      [*Metrica*],[*Descrizione*],[*Valore accettazione*],[*Valore ideale*],
+      ..fornitura.flatten()  
+)
+
+=== Sviluppo 
+
+
+==== Progettazione architteturale 
+
+Sono state individuate 2 metriche principali:
+
+            + *SFIN*: _Structure Fan In_;
+                  - *_Descrizione_*: Rappresenta il numero di moduli o componenti che chiamano un modulo o una funzione specifica;
+                  - *_Significato_*: Un fan-in elevato indica che molte parti del sistema dipendono da un particolare modulo. Questo può essere un segno positivo di riusabilità, in quanto il modulo è utilizzato in molte parti del sistema.
+            + *SFOUT*: _Structure Fan Out_;
+                  - *_Descrizione_*:rappresenta il numero di moduli o funzioni chiamati da un modulo o una funzione specifica.;
+                  - *_Significato_*:Un fan-out elevato può indicare che un modulo ha molte dipendenze da altri moduli. Questo può portare a una maggiore complessità del sistema, poiché le modifiche in un modulo possono richiedere modifiche in molti altri moduli.
```

Comment:
- manca spazio tra termine in grassetto e testo a seguire;
-  funzione specifica.; --> funzione specifica;